src/components/Intro/Intro.module.css

```diff
@@ -0,0 +1,41 @@
+.wrapper {
+    max-width: 288px;
+    margin: 64px auto 0 auto;
+    display:flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.title {
+    display: inline-block;
+    font-family: "InterBlack";
+    color: white;
+    font-size: 32px;
+    line-height: 35px;
+}
+
+.Introtext {
+    margin-top: 40px;
+    font-family: "InterRegular";
+    color: white;
+    font-size: 16px;
+    line-height: 26.08px;
+}
+
+.Introlist {
+    display: inline-block;
+    list-style: none;
+
+}
+
+.Introitem {
+    padding-inline-start: 0px;
+    font-family: "InterRegular";
+    font-size: 12px;
+    line-height: 18px;
+    color: white;
+}
+
+.bolditem {
```

Comment:
use the pattern to name it like `introList` it will help you to `separateOneWorkFromEnotherIfTheClassnameEnormouslyBig`

Reply:
done

Reply:
It seems that you forgot to push changes